src/js/components/plans/preflightResults.tsx

```diff
@@ -41,13 +41,13 @@ export const JobError = ({ errors }: { errors: StepResult[] }) => {
   const errorList = [];
   const warnList = [];
   let listItem = null;
-  for (const err of errors) {
+  for (const [idx, err] of errors.entries()) {
```

Comment:
@jgerigmeyer does this change make it possible to see more than one error or warning message?

Reply:
No, I don't think this has any functional impact. I just noticed a warning in the console about missing a [unique `key` on each item in the list](https://reactjs.org/docs/lists-and-keys.html).